src/types.ts

```diff
@@ -27,37 +27,69 @@ export interface Instance {
     account: string,
     bid: Bid
   ) => Promise<boolean>;
-  isZAuctionApprovedToTransferNft: (account: string) => Promise<boolean>;
-  getZAuctionSpendAllowance: (account: string) => Promise<ethers.BigNumber>;
+  isZAuctionApprovedToTransferNft: (
+    account: string,
+    contractAddress: string
+  ) => Promise<boolean>;
+  isZAuctionApprovedToTransferNftLegacy: (
+    account: string,
+    contractAddress: string
+  ) => Promise<boolean>;
   getZAuctionSpendAllowanceByBid: (
     account: string,
     bid: Bid
   ) => Promise<ethers.BigNumber>;
-  getTradeTokenAddress: () => Promise<string>;
+  getZAuctionSpendAllowance: (
+    paymentTokenAddress: string,
+    account: string
+  ) => Promise<ethers.BigNumber>;
+  getZAuctionSpendAllowanceLegacy: (
+    account: string
+  ) => Promise<ethers.BigNumber>;
+  getDefaultPaymentToken: () => Promise<string>;
+  setDefaultPaymentToken: (
+    defaultTokenAddress: string,
+    signer: ethers.Signer
+  ) => Promise<ethers.ContractTransaction>;
+  getDomainNetworkPaymentToken: (networkId: string) => Promise<string>;
+  setDomainNetworkPaymentToken: (
+    networkId: string,
+    domainNetworkToken: string,
+    signer: ethers.Signer
+  ) => Promise<ethers.ContractTransaction>;
+  getPaymentTokenForDomain: (domainTokenId: string) => Promise<string>;
```

Comment:
Everywhere else we usually just call it `tokenId` can we call this `tokenId` too?

Reply:
In the other PR you mentioned that there are a lot of "tokens" floating around, and it would be best to specify which token is an ERC20 "payment token" and which is an NFT "domain token", is that different here?

Reply:
That's fair, but I'm concerned about consistency with the rest of the SDK. Technically they are all domain tokens, so `domainTokenId` is a better name, but then we should change it everywhere.

Reply:
Seems more prudent right now to leave it alone, so I will adjust to your original comment